specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
+- Members can start posting messages to the Community board.
+- Members can perform a sibling split along one of the three dimensions
+- Members can perform a child split along one of the three dimensions, 
+- Member executing the split will become the owner of the new Community
+
+### Phase 2: Admin Functions
+- Admins can assign other admins
+- Users can request core memberships
+- Admins can approve core memberships
+- Admins can warn, silence and ban users
+
+### Phase 3: Enable voting
+- Members can request a vote on splitting the Community
+- Members can launch simple EasyPolls
+- Members can vote in a new core user
+- Members can vote to warn / silence / ban an offending user
+
+
 ## Use Cases
 
-### Actors
+## Actors
+<!-- List all collections that will use the widget. -->
+- Members
+   - Admin members
+   - Core members
+   - Home members
+   - Hangaround members
+- Non-members
+   - Non-member humans
+   - Non-member users
+
+## Actions
+<!-- List the actions each collective will take individually. -->
+
+### Find and Join a Community
+Finding a community aligned with my interests should be very easy. It's a core concept of this widget and serves to get people into open and collaborative communities rather than creating competing factions.
+
+Searching for communities should be possible through a few different ways
+- Search by map. Zoom into a map and update the list of available communities accordingly
+- Seach-as-you-type.
+   - Search by fuzzy search of geographies. Start typing in a city, country or region and all available options are shown. 
+   - Search by fuzzy search of constellations. Start typing in a constellation and all available options are shown.
+- Select you language, which updates the available communities. Default language is English.
+- Any combination of the above will further narrow the search
+
+
+### Approve a new Community member
+Approvind a new community member could be as simple as a "follow back" by the owner of the community or someone who has been given the keys to control the community account. 
+
+It is recommended that only human-verified accounts be allowed to become members, as a minimum requirement.
+
+A drawback of this approach is that there is no vote. Any single admin can follow or unfollow, which may be too arbitrary for some communities. Another drawback is that the "follow" feature would no longer be available to use for its originally intended purpose. A more structured approach to membership may have to be contemplated. Optins include:
+- user follows the Community and automatically becomes a member as long as some criteria are filled (e.g. is human verified and accepts the code of conduct) => Preferred
+- user applies for membership, followed by a majority vote to approve the application
+- membership is open to anyone without approval
+- membership is open to anyone, with the restriction that they can be a full member of one and only one community
```

Comment:
Why such restrictive requirement? I may be a member of Swahili speakers community, and also member of ReFi community.

Reply:
Not restrictive. These are different membership types, all of which could be available in one community. Most communities would allow multiple types of memberships (all enabled by default). Of course you can be member of many communities. 

I was saying there could be one membership type whereby you have to pick your one and only "home" community. But maybe this is not a membership type as much as it is a flag.